lib/db.js

```diff
@@ -38,13 +38,13 @@ export default (connection_string) => {
       )
       .catch(err => {
         const { message, position } = err;
-        if (message && position >= 0) {
+        if (message && position >= 1) {
           const endLineWrapIndexOf = string.indexOf('\n', position);
           const endLineWrapPos = endLineWrapIndexOf >= 0 ? endLineWrapIndexOf : string.length;
           const stringStart = string.substring(0, endLineWrapPos);
           const stringEnd = string.substr(endLineWrapPos);
           const startLineWrapPos = stringStart.lastIndexOf('\n') + 1;
```

Comment:
It is ```
const startLineIndexOf = stringStart.lastIndexOf('\n');
const startLineWrapPos = startLineIndexOf >= 0 ? startLineIndexOf + 1 : 0;```
Because we want to start AFTER `\n` or on first character...